release/gcb/generate_manifest.sh

```diff
@@ -59,15 +59,19 @@ function create_manifest_check_consistency() {
     echo "ISTIO_REPO_SHA:$ISTIO_REPO_SHA API_REPO_SHA:$API_REPO_SHA PROXY_REPO_SHA:$PROXY_REPO_SHA CNI_REPO_SHA:$CNI_REPO_SHA some shas not found"
     exit 8
   fi
+  popd
+  checkout_code "api" "${API_REPO_SHA}" .
+  pushd api || exit
+    API_SHA=$(git rev-parse "${API_REPO_SHA}")
+  popd
```

Comment:
should we check this git rev-parse command error return somehow? (to detect if SHA is malformed or doesn't exist or something else?)

Reply:
Wouldn't be bad, but either way we would see the errors in the logs.  So the error check would really just make it easier to debug a failure